db/migrations/20210111171826_create_licences.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const tableName = 'licences'
+
+exports.up = async function (knex) {
+  await knex
+    .schema
+    .createTable(tableName, table => {
+      // Primary Key
+      table.uuid('id').primary().defaultTo(knex.raw('gen_random_uuid()'))
+
+      // Data
+      table.uuid('invoice_id').notNullable()
```

Comment:
I think it would be worth storing the bill run ID directly as well in the table. That way we have a direct link between the `bill_runs` and `licences` and don't have to go via `invoices. What do you think?